File: 2_Programação_sistemas_operacionais/2.1_File_stdio/Python/Ex6.py
def abre_arq(arquivo, modo):
	try:
		p = open(arquivo, modo)
	except OSError:
		print("Erro! Impossivel abrir o arquivo!")
		exit(-1)
	return p

fp = abre_arq("arquivo.txt", "w")
string = "0"
while(len(string) > 0):
	print("Digite uma nova string. Para terminar, digite <enter>: ")
	string = input()
	fp.write(string + '\n')
fp.close()

File: 2_Programação_sistemas_operacionais/2.1_File_stdio/Ex6.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include "file_stdio.h"

int main()
{
	FILE *fp;
	char string[100];
	fp = abre_arq("arquivo.txt", "w");
	do
	{
		printf("\nDigite uma nova string."
			" Para terminar, digite <enter>: ");
		gets(string);
		fputs(string, fp);
		putc('\n', fp);
	} while(strlen(string) > 0);
	fclose(fp);
}
